src/__configuration__/themes/halloween.ts

```diff
@@ -71,3 +77,23 @@ export const HalloweenTheme = {
         },
     },
 };
+
+export const HalloweenSchedule: Schedule = {
+    start: new Date(0, 9, 14), // Oct 14
+    end: new Date(0, 10, 13), // Nov 13
```

Comment:
2 weeks after halloween seems like a lot. Maybe 1?

Reply:
Changing to Nov 2... Don't feel like doing more Halloween things after the Election day

Reply:
Election Day theme!